GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -2944,76 +2944,72 @@ private void copyFolderNameMenuItem_Click(object sender, EventArgs e)
             FilenameToClipboardToolStripMenuItemClick(sender, e);
         }
 
-        private void commitTemplatesConfigtoolStripMenuItem_Click(object sender, EventArgs e)
+        private void LoadCommitTemplates()
         {
-            using (var frm = new FormCommitTemplateSettings())
+            commitTemplatesToolStripMenuItem.DropDownItems.Clear();
+
+            // Add registered templates
+            foreach (var item in _commitTemplateManager.RegisteredTemplates)
             {
-                frm.ShowDialog(this);
+                CreateToolStripItem(item);
             }
 
-            _shouldReloadCommitTemplates = true;
-        }
+            AddSeparator();
 
-        private void LoadCommitTemplates()
```

Comment:
My head spins trying to find how the old code worked, but the new seem logical...